src/inc/palclr.h

```diff
@@ -484,7 +484,14 @@
 //  and it provides __declspec(selectany) to instruct the linker to merge
 //  duplicate external const static data copies into one.
 //  
+#if defined(__clang__)
+#define SELECTANY extern
```

Comment:
@janvorli I could introduce a define SOURCE_FORMATTING and change the ifdef to use that. Would that be better?